common/Environments/Models/CreateComputeSystemOperation.cs

```diff
@@ -117,6 +124,19 @@ public void StartOperation()
                 Completed?.Invoke(this, CreateComputeSystemResult);
             }
 
+            var completionStatus = EnvironmentsTelemetryStatus.Succeeded;
+
+            if ((CreateComputeSystemResult == null) || (CreateComputeSystemResult.Result.Status == ProviderOperationStatus.Failure))
+            {
+                completionStatus = EnvironmentsTelemetryStatus.Failed;
+                LogFailure(CreateComputeSystemResult);
+            }
+
+            TelemetryFactory.Get<ITelemetry>().Log(
```

Comment:
It could be difficult to correlate this event against Started above without using activity ID. We'll have thread ID, but this event could be lost or not fired due to an exception and the another Start operation could fire these events from the same thread.
TelemetryFactory creates a singleton that has an activity ID, but it's the same ID for all calls to that instance of Telemetry class (that could be used to identify events from the same instance of Dev Home).
Consider using relatedActivityId parameter to make correlation of these events easier.